.github/workflows/build-hw-reusable.yml

```diff
@@ -82,6 +82,7 @@ jobs:
         -DUR_ENABLE_TRACING=ON
         -DUR_DEVELOPER_MODE=ON
         -DUR_BUILD_TESTS=ON
+        -DUR_CONFORMANCE_TEST_LOADER=OFF
```

Comment:
`UR_CONFORMANCE_TEST_LOADER` should be added to the list of flags in `README.md`